scripts/inspect.sh

```diff
@@ -267,7 +267,7 @@ function suggest_fixes {
   fi
 
   if ! grep -q 'cgroup/memory' /proc/self/mountinfo; then
-    if ! grep -q 'cgroup/unified.* cgroup2 ' /proc/self/mountinfo; then
+    if ! grep -q 'cgroup2' /proc/self/mountinfo; then
```

Comment:
The check that you replace here used to work at some point and it probably still valid for older systems. Could we keep both checks?